internal/operators/ocs/consts.go

```diff
@@ -3,10 +3,14 @@ package ocs
 type ocsDeploymentMode string
 
 const (
-	ssdDrive     string            = "SSD"
-	hddDrive     string            = "HDD"
-	minDiskSize  int64             = 5 //5GB is the min disk size for OCS
-	compactMode  ocsDeploymentMode = "Compact"
-	minimalMode  ocsDeploymentMode = "Minimal"
-	standardMode ocsDeploymentMode = "Standard"
+	CPUCompactMode      int64             = 6  // per host requires 6 cpus for OCS in compact mode
+	CPUMinimalMode      int64             = 4  // per host requires at least 4 cpus for OCS in minimal mode
+	MemoryGBCompactMode int64             = 19 // per host requires at least 19 GB RAM for OCS in compact mode
+	MemoryGBMinimalMode int64             = 11 // per host requires at least 11 GB RAM for OCS in minimal mode
```

Comment:
There is no change in memory requirements between standard and minimal mode. Why is this different here?